src/components/AccountCard.tsx

```
@@ -0,0 +1,87 @@
+interface SourceAccountProps {
+  cardData: CardData[];
+  selectedCard: CardData | undefined;
+  setSelectedCard: React.Dispatch<React.SetStateAction<CardData | undefined>>;
+}
+
+const AccountCard: React.FC<SourceAccountProps> = ({
+  cardData,
+  selectedCard,
+  // setSelectedCard,
+}) => {
+  const [currentIndex, setCurrentIndex] = useState(0);
```

ESLint

[error] 'currentIndex' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)